users/bescka/floating_point_arithmetic_web.html

```diff
@@ -0,0 +1,366 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Flounders and Codes - Home</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <link rel="icon" href="/assets/fc_favicon_96_2.ico" type="image/x-icon">
+    <link rel="shortcut icon" href="/assets/fc_favicon_96_2.ico" type="image/x-icon">
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
+    <script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <div class="container">
+        <div class="ascii-art">
+            - ~- ~ - - O o <br>
+            &nbsp; ~- - ~ - o -<br>
+            ~ - &gt;&lt;(((º&gt; ~ -<br>
+            &nbsp; - ~ - - -- ~ <br>
+        </div>
+        <span class="title_container">
```

Comment:
You could make the page header into a link back to the home page

Reply:
as above